lib/version.js

```diff
@@ -1,3 +1,4 @@
-"use strict";
 
-module.exports = "1.1.0";
+'use strict';
+
+module.exports = '1.2.0';
```

Comment:
Lint violation + that should be changed on release
```suggestion
"use strict";

module.exports = "1.1.0";
```

Reply:
Lint violation - crimeny there's 1400 of them

forgot to add linting to the build.  fixing 💔 

Version - this is a generated file, so whatever we decide elsewhere will carry here too

Reply:
Quotes are now fixed.  The other topic is a direct descendant of previous comments and either way this step will not change